CMakeLists.txt

```diff
@@ -80,6 +93,23 @@ if(BUILD_EXAMPLES)
     add_subdirectory(examples)
 endif()
 
+# ==================================================================
+# CMake Summary
+# ==================================================================
+
+message("")
+message("-- KokkosFFT version: ${KOKKOSFFT_VERSION_MAJOR}.${KOKKOSFFT_VERSION_MINOR}.${KOKKOSFFT_VERSION_PATCH}")
+message("-- KokkosFFT TPLs")
```

Comment:
The `-- ` prefix is rendered by using the `STATUS` mode (see [message](https://cmake.org/cmake/help/latest/command/message.html) documentation:

```cmake
message(STATUS "KokkosFFT version: ${KOKKOSFFT_VERSION_MAJOR}.${KOKKOSFFT_VERSION_MINOR}.${KOKKOSFFT_VERSION_PATCH}")
message(STATUS "KokkosFFT TPLs")
```